### PR TITLE
Add a update production command

### PR DESCRIPTION
**Pull request by Lyxal**

### CODE REVIEW

File: hooks/msghooks.py [hunk 1]
```diff
@@ -246,8 +246,53 @@ def receive_message():
                     + r.json()["message"]
                 )
             return ""
+        
         if re.match(r"^issue", without_ping):
             return reply + ISSUE_HELP
+        
+        if re.match(r"(update )?prod(uction)?", without_ping):
```
Comment (Steffan153) [inline hooks/msghooks.py:253]: you should probably add a `^` to the beginning of the regex

**Review by Steffan153 — CHANGES_REQUESTED** [review]

**Review by hyper-neutrino — APPROVED** [review]

**Review by Steffan153 — APPROVED** [review]

File: hooks/msghooks.py [hunk 2]
```diff
@@ -246,8 +246,53 @@ def receive_message():
...
             return ""
+        
         if re.match(r"^issue", without_ping):
             return reply + ISSUE_HELP
+        
+        if re.match(r"^(update )?prod(uction)?", without_ping):
+            # Create a PR for main -> production
+            # but first check for privleges
+            
+            if message["user_id"] not in STORAGE["privileged"]:
```
Comment (cgccuser) [inline hooks/msghooks.py:257]: Maybe this check could be extracted to a helper method

**Review by cgccuser — APPROVED** [review]
I don't understand this very well but I'll approve anyway because it won't go past otherwise.

File: hooks/msghooks.py (hunk 2 above)
Comment (hyper-neutrino) [reply hooks/msghooks.py:257]: I will eventually rework the entire bot to improve the structure and remove duplication. Alternatively, we could PR this separately to fix some similar issues as well.

**Pull request merged**